static/scripts/PlotHandler.js

```diff
@@ -0,0 +1,121 @@
+import { addNewPlotEntry } from "./workspace";
+
+class PlotHandler {
+    constructor() {
+        this.plotMap = new Map();
+    }
+    //the data forwarded contains the information: data(yValue,datasetName,plotName,plotType)
+    //forward the data to the correct PlotWorker or create one if necessary 
+    updateValue(data) {
+        let plotName = data[3];
+        let requestedPlot = this.plotMap.get(plotName);
+        if (requestedPlot == undefined) {
+            requestedPlot = new PlotWorker(plotName, data[4]);
+            this.plotMap.set(plotName, requestedPlot);
+        }
+        requestedPlot.updateValue(data);
+    }
+
+    drawPlots() {
```

Comment:
Does this provide the functionality to update the plots in realtime?

Reply:
In theory yes. However, it doesn't work really well. I am planning to create a seperate PR that implements proper realtime plot creation.